packages/react-flex/src/components/Flex/useFlexStyles.ts

```diff
@@ -16,11 +25,22 @@ export const useFlexStyles = (state: FlexState): FlexState => {
   const styles = useStyles();
   state.className = mergeClasses(styles.root, state.className);
 
-  // TODO: Hook up slot styles, for example:
-  // const exampleSlotClassName = useExampleSlotStyles(state);
-  // if (state.exampleSlot) {
-  //   state.exampleSlot.className = mergeClasses(exampleSlotClassName, state.exampleSlot.className);
-  // }
+  const direction = state.direction ?? 'row';
+  const horizontalAlign = state.horizontalAlign ?? 'normal';
+  const verticalAlign = state.verticalAlign ?? 'normal';
+
+  state.style = {
```

Comment:
Perhaps wrap this with `useMemo` for performance ?

Reply:
I'm not sure `useMemo` will bring us advantages vs the tradeoffs, given we'd have to add basically all the props of `state` to the dependencies. IMO we should go with this and worry about performance if any issues arise as synthetic vs real use performance will be vastly different. What do you think?

Reply:
hmm, this would need to add every property that is being used as the dependency -> increasing memory for unknown benefit for now.

Reply:
That's fair, can disregard then